build-scripts/docker_build.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -x
+set -e
+set -a
+
+WLRM_FOLDER=stable-$(echo $NGINX_VERSION | sed 's/\.//g')
+
+# Single-platform for local, multi-platform for CI
+if [ ! -z ${CI+x} ]; then
+  PLATFORMS=${PLATFORMS:-linux/amd64,linux/aarch64}
+  ARCHS=${ARCHS:-x86_64 aarch64}
+  BUILDX_ARG={BUILDX_ARGS:---push}
+else
+  ARCH=$(uname -m)
+  if [ "$ARCH" = "x86_64" ]; then
+    PLATFORMS=${PLATFORMS:-linux/amd64}
```

Comment:
The variable PLATFORMS , ARCH and BUILDX_ARGS can not be set elsewhere except this script, so we do not need to have their defaults here